include/core/io/platform/f3xx/ADCf3xx.hpp

```diff
@@ -8,14 +8,18 @@
 
 namespace core::io {
 
+enum class ADCPeriph {
+    ONE
+};
+
 class ADCf3xx : public ADC {
 public:
     /**
      * Setup the given pin for ADC usage
      *
      * @param[in] pin The pin to setup for ADC
      */
-    ADCf3xx(Pin pin);
+    ADCf3xx(Pin pin, ADCPeriph adcPeriph);
```

Comment:
Default value?

Reply:
Default value is set in the template thingy in manager for adc